### PR TITLE
Added DB defines and Header for SceUsbSerial

### PR DESCRIPTION
**Pull request by geekbozu**
So I have no idea how you guys actually want function names styled and such. So I tried my best. Let me know what I can change to make it fit better!

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -5523,5 +5523,39 @@
...
+		"nid": 2284438259,
+		"variables": {},
+		"functions":{
+			"SceUsbSerialStartForDriver": 4276614330,
+			"SceUsbSerialSetupForDriver": 1493929879,
+			"SceUsbSerialStatusForDriver": 2353162502,
+			"SceUsbSerialCloseForDriver": 3870418511,
+			"SceUsbSerialget_lenForDriver": 4113675694,
+			"SceUsbSerialSendForDriver": 204370880,
+			"SceUsbSerialrecvForDriver": 1801333103
```
Comment (d3m3vilurr) [inline db.json]: functions should start `sce` instead `Sce`

File: include/psp2/usbser.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+	
+int SceUsbSerialStartForDriver(void);
+int SceUsbSerialSetupForDriver(int unk);	
+int SceUsbSerialStatysForDriver(void);
+int SceUsbSerialCloseForDriver(void);
+unsigned int SceUsbSerialget_lenForDriver(void);
+unsigned int SceUsbSerialSebdForDriver(const void *buffer, unsigned int len, int unk1, int unk2);	
+unsigned int SceUsbSerialrecvForDriver(void *buffer, unsigned int max_len, int unk1, int unk2);
```
Comment (d3m3vilurr) [inline include/psp2/usbser.h]: can you add function documents like other header files?

File: db.json (hunk 1 above)
Comment (geekbozu) [inline db.json]: sounds good. Also seems like I didn't follow my own naming convention. Whoops!

SceUsbSer
SceUsbSerialForDriver

Doh!

File: include/psp2/usbser.h (hunk 2 above)
Comment (geekbozu) [inline include/psp2/usbser.h]: I realized after I made the request i forgot them. Will add them tommorow after work!

File: db.json (hunk 1 above)
Comment (d3m3vilurr) [inline db.json]: we don't know original sdk rules, and db file just defined function-name map.
so we can use own naming rule (well... maybe related #80)

current project convention is function name should be `camelCase`.
i guess, you should change function name in both db & header  :)

File: include/psp2/usbser.h [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#endif
+
+
+
+	
+int SceUsbSerialStartForDriver(void);
+int SceUsbSerialSetupForDriver(int unk);	
+int SceUsbSerialStatysForDriver(void);
+int SceUsbSerialCloseForDriver(void);
+unsigned int SceUsbSerialget_lenForDriver(void);
```
Comment (frangarcj) [inline include/psp2/usbser.h]: Better sceUsbSerialGetLenForDriver ?

File: include/psp2/usbser.h [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+	
+int SceUsbSerialStart(void);
+int SceUsbSerialSetup(int unk);	
+int SceUsbSerialClose(void);
+int SceUsbSerialStatus(void);
+unsigned int SceUsbSerialget_len(void);
+unsigned int SceUsbSerialSend(const void *buffer, unsigned int len, int unk1, int unk2);	
+unsigned int SceUsbSerialrecv(void *buffer, unsigned int max_len, int unk1, int unk2);
+#endif
```
Comment (d3m3vilurr) [inline include/psp2/usbser.h]: missing 
```
#ifdef __cplusplus
}
#endif
```
:)

File: include/psp2/usbser.h [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+#include <psp2/types.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+
```
Comment (d3m3vilurr) [inline include/psp2/usbser.h]: please remove multiple blanks

File: include/psp2/usbser.h [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
```
Comment (d3m3vilurr) [inline include/psp2/usbser.h]: Please add license header

File: db.json [hunk 7]
```diff
@@ -5523,5 +5523,39 @@
                 }
             }
         }
+    },
+    "SceUsbSer":{
```
Comment (d3m3vilurr) [inline db.json]: i guess SceUsbSerial is more good then this

File: include/psp2/usbser.h (hunk 4 above)
Comment (d3m3vilurr) [inline include/psp2/usbser.h]: also filename `usbser.h` maybe change to `usbserial.h`

**Pull request merged**